build.gradle

```diff
@@ -98,6 +99,23 @@ subprojects {
   plugins.withType(BuildPlugin).whenPluginAdded {
     project.licenseFile = project.rootProject.file('licenses/APACHE-LICENSE-2.0.txt')
     project.noticeFile = project.rootProject.file('NOTICE.txt')
+
+    // This plugin complains if the java plugin hasn't been applied yet, so
+    // we don't have this task at the top-level.
+    project.apply plugin: "com.diffplug.gradle.spotless"
+
+    spotless {
+      java {
+        googleJavaFormat().aosp()
+        // Required for :modules:lang-painless: - avoids a rule failing to
+        // make up its mind about formatting.
+        paddedCell()
+      }
+    }
+
+    project.afterEvaluate {
```

Comment:
The after evaluate should not be needed here as `precommit` should exist at this time and the argument to `dependsOn` is a string that gets resolved lather.